website/src/pages/plugins/other/introspection.mdx

```diff
@@ -2,10 +2,9 @@
 title: introspection
 ---
 
-import { PackageApiDocs, PackageHeader } from '@/components'
-import { getNpmInfo } from '@/lib/get-npm-info'
+import { PluginApiDocs, PluginHeader } from '@/components/plugin'
```

Comment:
We'll need to update the Algolia script to remove those import lines from the content before indexing.
I added it to the list of search improvements: https://github.com/the-guild-org/the-guild-components/issues/372